assets/javascripts/discourse/initializers/extend-for-kolide.js.es6

```diff
@@ -0,0 +1,24 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+import I18n from "I18n";
+
+function initializeWithApi(api) {
+  const currentUser = api.getCurrentUser();
+
+  if (currentUser) {
+    const site = api.container.lookup("site:main");
+
+    if (site.non_onboarded_device) {
+      api.addGlobalNotice(
+        I18n.t("discourse_kolide.non_onboarded_device.notice", { link: "/t/" }),
```

Comment:
hmmm I guess we accept a topic id in a site setting for onboarding instructions for now?